cuda_pathfinder/tests/test_load_nvidia_dynamic_lib.py

```diff
@@ -90,6 +93,12 @@ def _get_libnames_for_test_load_nvidia_dynamic_lib():
     return tuple(result)
 
 
+def _quote_for_shell_copypaste(s):
+    if os.name == "nt":
+        return list2cmdline([s])
+    return shlex.quote(s)
```

Comment:
It's really awful that this hasn't been solved upstream in the standard library.

Reply:
Same thought last night! And on top of that the skirmish with our tooling (bandit `# nosec B404`). See other comment for resolution.